GameData/ROEngines/PartConfigs/GEM40_BDB.cfg

```diff
@@ -0,0 +1,120 @@
+PART
+{
+	module = Part
+	name = ROE-GEM40
+	author = Cobaltwolf, Vader111
+	
+	category = Engine
+	subcategory = 0
+	RSSROConfig = true
+	RP0conf = true
+	
+	mass = 1.1879
+	crashTolerance = 10
+	maxTemp = 673.15
+	skinTemp = 773.15
+	fuelCrossfeed = true
+	breakingForce = 10000
+	breakingTorque = 10000
+	
+	//	============================================================================
+	//	Update Below
+	//	============================================================================
+	
+	MODEL
+	{
+		model = ROEngines/Assets/BDB/Delta/bluedog_Delta_GEM40
+	}
+	
+	scale = 1.0
+	rescaleFactor = 1.6
+	node_attach = 0.0, 0.0, -0.3125, 0.0, 0.0, 1.0, 1
+	CoMOffset = 0.0, -0.16, -0.16
+	// stack, srfAttach, allowStack, allowSrfAttach, allowCollision
+	attachRules = 1,1,1,1,0	
+	
+	title = GEM 40
+	manufacturer = Hercules
+	description = The Graphite-Epoxy Motor (GEM) replaced the steel case used on earlier Castor-series boosters with a lighter composite case. The 40-inch (1-meter) diameter GEM 40 was used on the Delta II 7000-series in sets of three, four, or nine. When nine boosters were used, six were ignited at liftoff and the remaining three were ignited after burnout and jettison of the first six. Burn time: 58 seconds.
+	
+	tags = ATK, GEM40, booster, radial, solid, srb,
+	
+	engineType = GEM-40
+	stagingIcon = SOLID_BOOSTER
+	stageOffset = 1
+	childStageOffset = 1
+	
+	skinInternalConductionMult = 4.0
+	emissiveConstant = 0.5
+	dragModelType = default
+	maximum_drag = 0.3
+	minimum_drag = 0.2
+	angularDrag = 2
+	breakingForce = 200
+	breakingTorque = 200
+	crashTolerance = 7
+	
+	TechRequired = unlockParts
+	cost = 150
+	entryCost = 1
```

Comment:
Make use of ROESetEngineDefaults (also duplicate decls for the breaking force/torque)